src/neo4j-arc/graph-visualization/GraphVisualizer/Graph/visualization/Visualization.ts

```diff
@@ -132,40 +132,29 @@ export class Visualization {
           .call(sel => (isZoomClick ? sel.ease(easeCubic) : sel))
           .attr('transform', String(e.transform))
       })
-
-    const zoomEventHandler = (
-      selection: Selection<SVGElement, unknown, BaseType, unknown>
-    ) => {
-      const handleZoomOnShiftScroll = (e: WheelEvent) => {
-        const modKeySelected = e.metaKey || e.ctrlKey || e.shiftKey
-        if (modKeySelected || !this.wheelZoomRequiresModKey) {
-          e.preventDefault()
-
-          // This is the default implementation of wheelDelta function in d3-zoom v3.0.0
-          // For some reasons typescript complains when trying to get it by calling zoomBehaviour.wheelDelta() instead
-          // but it should be the same (and indeed it works at runtime).
-          // https://github.com/d3/d3-zoom/blob/1bccd3fd56ea24e9658bd7e7c24e9b89410c8967/README.md#zoom_wheelDelta
-          const delta =
-            -e.deltaY * (e.deltaMode === 1 ? 0.05 : e.deltaMode ? 1 : 0.002)
-
-          return this.zoomBehavior.scaleBy(this.root, 1 + delta)
-        } else {
-          onDisplayZoomWheelInfoMessage()
+      // This is the default implementation of wheelDelta function in d3-zoom v3.0.0
```

Comment:
Could be worth mentioning why we kept this (aka the ctrl being to fast otherwise)